instrumentation/github.com/gin-gonic/gin/otelgin/test/gintrace_test.go

```diff
@@ -17,14 +17,14 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin"
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/codes"
 	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 	"go.opentelemetry.io/otel/sdk/trace/tracetest"
-
-	"go.opentelemetry.io/otel/attribute"
 	oteltrace "go.opentelemetry.io/otel/trace"
+
+	"go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin"
```

Comment:
Can you revert this import rearrange change? We are grouping `go.openteletry.io`. See https://github.com/open-telemetry/opentelemetry-go-contrib/blob/96b1ed1e7b0c1613290df1801ff6ba8474a33ce5/.golangci.yml#L108-L109

Reply:
I'll handle it together later.

Reply:
Reopened 😉